saleor/payment/utils.py

```diff
@@ -495,3 +517,28 @@ def gateway_refund(payment, amount: Decimal = None) -> Transaction:
 
     _gateway_postprocess(transaction, payment)
     return transaction
+
+
+def extract_id_for_payment_gateway(user, gateway):
+    key = prepare_label_name(gateway)
+    gateway_config = user.get_private_meta(label=GATEWAYS_LABEL).get(key, {})
+    return gateway_config.get("customer_id", None)
+
+
+def store_id_for_payment_gateway(user, gateway, customer_id):
```

Comment:
This function could use some docstring - when you look at the usage of it in a mutation it's hard to guess what ID it stores. Or shouldn't it be called `store_customer_id`?